src/locales/en/translations.json

```diff
@@ -4,21 +4,7 @@
     "services": {
       "title": "Development & Design",
       "topline": "Our Services",
-      "text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements im Konzerngeschäft. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen:",
-      "teasers": {
-        "first": {
-          "title": "Enterprise Applications & Libraries",
-          "text": "Teams aus Designern, Produktmanagern, Entwicklern, Supportmitarbeitern, Marketingspezialisten usw. bringen die richtige Mischung aus Know-how und Erfahrung mit und sorgen dafür, dass auch mal über den Tellerrand geschaut wird."
-        },
-        "second": {
-          "title": "Technical Analysis",
-          "text": "Jedes Team bei Satellytes ist für seine eigenen Aufgaben verantwortlich. In den Worten des Agilen Manifests: „Die besten Architekturen, Anforderungen und Entwürfe entstehen durch selbstorganisierte Teams.“"
-        },
-        "third": {
-          "title": "UX/UI",
-          "text": "Schnelles Feedback, kurze Entscheidungswege und Offenheit für Veränderung sind uns wichtig – egal, ob man es Scrum, Kanban oder XP nennt."
-        }
-      }
```

Comment:
I just used the keys from the service page because the content was the same. Even when it's different I would co locate it with the rest of the service relevant texts but provide a postfix `main` instead of grouping it under `main`. Much easier for the brain and we don't forget things we change in once place (which almost happened here)